integration/spring-pulsar/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'java'
+    id 'org.springframework.boot'
+    id 'org.springframework.boot.aot'
+    id 'org.springframework.aot.smoke-test'
+    id 'org.graalvm.buildtools.native'
+}
+
+repositories {
+    mavenLocal()
+    maven {
```

Comment:
It may be helpful to add a comment as to why this repo is here (backing reactive client).